locales/en/common.json

```diff
@@ -245,5 +245,8 @@
   "error-500": "500",
   "internal-server-error": "Internal Server Error !",
   "unable-to-find": "We're unable to find out what's happening! We suggest you to",
-  "try-again-later": "or visit here later."
+  "try-again-later": "or visit here later.",
+  "something-went-wrong": "Something went wrong",
```

Comment:
@ukrocks007  i am a little confuse if i use "toast.error(t(json?.error?.message || 'Something went wrong'));" then it will not translating the "Something went wrong" as it checks if the error is undefined.

also the part for "instead of using something-went-wrong let's use Something went wrong as key"  i dont understand. Are you saying i remove the - because it will not conform to the json file style.

also the required changes in green with the + already exist on line 248 and 249.

What i think you mean with "toast.error(t(json?.error?.message || 'Something went wrong'));" is to account for the undefined errors, we can add them into the code with 
"toast.error(
  json?.error?.message
  ? t('something-went-wrong')
    : json.error.message === 'Something went wrong'
  ? t('something-went-wrong')
    : json.error.message
);
" 
PS: Please advice

Reply:
`json?.error?.message || 'Something went wrong'` will return the error.message if it exists otherwise it will result in `Something went wrong`
When you change the key in translations `something-went-wrong` to `Something went wrong` the following code will work as expected,
```
toast.error(t(json?.error?.message || 'Something went wrong'));
```

So can you make these changes and test to check if it's working as expected?